pointcloud/basic_octree.go

```diff
@@ -26,9 +32,10 @@ type NodeType uint8
 // and serialization.
 type BasicOctree struct {
 	node       basicOctreeNode
-	center     r3.Vector
+	center     spatialmath.Pose
 	sideLength float64
 	size       int
+	label      string
```

Comment:
add label to node not octree struct. This is reserved for high level basic octree vars